### PR TITLE
upgrade everything, fingers crossed

### PR DESCRIPTION
**Pull request by antonina-cherednichenko**

### CODE REVIEW

**Review by jflatow — DISMISSED** [review]
🤞

File: spec/scenario/InKindLiquidation.scen [hunk 1]
```diff
@@ -237,7 +237,7 @@ Test "When price oracle for collateral token is whack"
     Assert Equal (Comptroller Liquidity Geoff) -1e18
     -- Okay, so we should be able to liquidate, so let's do that.
     Prep Torrey 10e18 BAT cBAT
-    PriceOracle SetPrice cBAT UInt256Max
+    PriceOracle SetPrice cBAT 1.15792e59
```
Comment (jflatow) [inline spec/scenario/InKindLiquidation.scen:240]: Why did this one change actually?

Comment (antonina-cherednichenko) [reply spec/scenario/InKindLiquidation.scen:240]: SetPrice multiplies the value by 1e18, so it's becoming out of range 
Step 1 - https://github.com/compound-finance/compound-protocol/blob/toni/upgarde-everything-fingers-crossed/scenario/src/Event/PriceOracleEvent.ts#L107 --> 
Step 2 https://github.com/compound-finance/compound-protocol/blob/toni/upgarde-everything-fingers-crossed/scenario/src/CoreValue.ts#L165

@jflatow is it okay ?

Comment (jflatow) [reply spec/scenario/InKindLiquidation.scen:240]: Ah I guess, its just setting up liquidation so seems okay...but `SetPrice` didn't change in this PR did it? Or was it before actually using `UInt256Max * 1e18` and truncating?

**Comment by jflatow** [comment]
Is this ready to merge?

**Pull request closed without merging**